### PR TITLE
wc_pairingDelete & wc_sessionDelete

### PR DESCRIPTION
**Pull request by dukevantreta**
Adds descriptions for `wc_pairingDelete` and `wc_sessionDelete`.

### CODE REVIEW

**Comment by dukevantreta** [comment]
How should the `reason` parameter be used, from a UX point of view? What's the recommended practice, is it a human-readable string to display on the peer's side?

File: wc-rpc-methods.md [hunk 1]
```diff
@@ -22,6 +22,13 @@ error: {
 ### wc_pairingExtend
 
 ### wc_pairingDelete
+Used to inform the peer to close and delete a pairing. All associated sessions of the given pairing must also be deleted.
+```jsonc
+// wc_pairingDelete params
+{
+    "reason": string
+}
```
Comment (jakubuid) [inline wc-rpc-methods.md]:
```suggestion
// wc_pairingDelete params
{
    "reason": string
    "code": int
}
```
Why we removed code?

File: wc-rpc-methods.md [hunk 2]
```diff
@@ -38,6 +45,13 @@ error: {
 ### wc_sessionExtend
 
 ### wc_sessionDelete
+Used to inform the peer to close and delete a session.
+```jsonc
+// wc_sessionDelete params
+{
+    "reason": string
+}
```
Comment (jakubuid) [inline wc-rpc-methods.md:55]: Same

**Comment by jakubuid** [comment]
> How should the `reason` parameter be used, from a UX point of view? What's the recommended practice, is it a human-readable string to display on the peer's side?

Whenever the session delete event is received by the peer, we return it to the skd's customer and display this message to the user.

File: wc-rpc-methods.md (hunk 1 above)
Comment (dukevantreta) [reply wc-rpc-methods.md]: It didn't exist on the example payloads here: https://docs.walletconnect.com/2.0/protocol/client-communication
I'll be adding it, anyways

File: wc-rpc-methods.md (hunk 2 above)
Comment (dukevantreta) [reply wc-rpc-methods.md:55]: Done

**Pull request merged**